src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -110,7 +143,7 @@ public bool RemoveSuppressions(bool selectedErrorListEntriesOnly)
                 }
 
                 var project = solution.GetProject(diagnosticsByProject.Key);
-                if (project != null)
+                if (project != null && shouldFixInProject(project))
```

Comment:
@CyrusNajmabadi I am going to address your feedback in a separate change. We need this in asap to unblock insertion.